### PR TITLE
add git workshops for May, June & July

### PR DESCRIPTION
**Pull request by davidwyld**
Note that June is a 2 half day workshop. Does it need two event pages?

### CODE REVIEW

**Comment by bobturneruk** [comment]
> Note that June is a 2 half day workshop. Does it need two event pages?

I would say ideally not.

**Comment by ptheywood** [comment]
> Note that June is a 2 half day workshop. Does it need two event pages?

IIRC the website events data is only really designed with single-day events in mind with the sorting etc, so with a single event page it'll dissapear after the first day from the event list.

**Comment by bobturneruk** [comment]
That's OK I think - people coming will need to do both days, so it will serve no purpose being on the site after the first day has been run.

File: _events/workshop-2022-06-06-git-zero-hero.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "git & GitHub through GitKraken - from Zero to Hero!"
+category: training
+tags: collaboration, git, github, gitkraken
+permalink: /training/workshop/2022-06-06-git-zero-hero
+date: 2022-06-06
```
Comment (bobturneruk) [inline _events/workshop-2022-06-06-git-zero-hero.md:6]:
```suggestion
date: 2022-06-06
end-date: 2022-06-07
```

**Review by bobturneruk — APPROVED** [review]
I think OK with the proposed change.

File: _events/workshop-2022-06-06-git-zero-hero.md (hunk 1 above)
Comment (davidwyld) [reply _events/workshop-2022-06-06-git-zero-hero.md:6]: Ah sweet, thanks @bobturneruk

**Pull request merged**